tool/update_gold_output_files.dart

```diff
@@ -0,0 +1,52 @@
+import 'dart:io';
+
+import 'package:glob/glob.dart';
+import 'package:glob/list_local_fs.dart';
+import 'package:path/path.dart' as p;
+
+final sourceFixturesDir = Directory('test_fixtures/source_files');
+final goldsDir = Directory('test_fixtures/gold_output_files');
+
+/// A temporary directory used to generate new golds.
+final tmpSourceDir = Directory('test/tmp_test_files');
+
+/// Updates gold files in [goldsDir] by re-generating them from a copy of the files in [sourceFixturesDir].
+Future<void> main() async {
+  print('Copying files from ${sourceFixturesDir.path} to temporary directory ${tmpSourceDir.path}');
+  if (tmpSourceDir.existsSync()) tmpSourceDir.deleteSync(recursive: true);
+  tmpSourceDir.createSync(recursive: true);
+  for (final file in Glob(p.join(sourceFixturesDir.path, '**')).listSync().whereType<File>()) {
+    final newPath = p.join(tmpSourceDir.path, p.relative(file.path, from: sourceFixturesDir.path));
+    File(newPath).parent.createSync(recursive: true);
+    file.copySync(newPath);
+  }
+
+  print('Building files in temporary directory ${tmpSourceDir.path}');
```

Comment:
This is kind of hacky, but it's way better than our current workflow of manually updating them.